actions.yaml

```diff
@@ -62,4 +62,22 @@ restore:
       description: |
         A backup-id to identify the backup to restore. Format: backup-id=<string>.
   required:
-    - backup-id
\ No newline at end of file
+    - backup-id
+
+resume-upgrade:
+  description: Upgrade remaining units (after you manually verified that upgraded units are healthy).
+  
+force-upgrade:
+  description: |
+    Potential of *data loss* and *downtime*
+        
+    Force upgrade of this unit.
+    
+    Use to
+    - force incompatible upgrade and/or
+    - continue upgrade if 1+ upgraded units have non-active status
+  params:
+    ignore-lock:
```

Comment:
we can find something a bit more "user" oriented, perhaps: `apply-now` or `force-now` or something

Reply:
It could also be: `ignore-order/priority/turn` or similar, feel free to come up with something that's as meaningful to the end user as `lock` is to us charm developers